examples/todo-intermediate/test/acceptance/application.acceptance.ts

```diff
@@ -97,6 +105,103 @@ describe('Application', () => {
     throw new Error('No error was thrown!');
   });
 
+  it('creates a todoList', async () => {
+    const todoList = givenTodoList();
+    const response = await client
+      .post('/todo-lists')
+      .send(todoList)
+      .expect(200);
+    expect(response.body).to.containDeep(todoList);
+    const result = await todoListRepo.findById(response.body.id);
+    expect(result).to.containDeep(todoList);
+  });
+
+  it('counts todoLists', async () => {
+    const todoLists = [];
+    todoLists.push(await givenTodoListInstance());
+    todoLists.push(
+      await givenTodoListInstance({title: 'so many things to do wow'}),
+    );
+    const response = await client
+      .get('/todo-lists/count')
+      .send()
+      .expect(200);
+    expect(response.body).to.eql(todoLists.length);
+  });
+
+  it('finds all todoLists', async () => {
+    const todoLists = [];
+    todoLists.push(await givenTodoListInstance());
+    todoLists.push(
+      await givenTodoListInstance({title: 'so many things to do wow'}),
+    );
+    const response = await client
+      .get('/todo-lists')
+      .send()
+      .expect(200);
+    expect(response.body).to.containDeep(todoLists);
+  });
+
+  it.skip('updates all todoLists', async () => {
```

Comment:
Skipped for now since there's a bug (I think) with `updateAll` atm where an error is thrown when `updateAll(undefined, someData)` signature is used. See #1529